C:30:"PHPUnit\Runner\TestResultCache":1506:{a:2:{s:7:"defects";a:3:{s:120:"Tests\Core\Application\Service\GetUserBalanceUseCaseTest::testThatIfTheUserIsPresentThenAnSpecificResponseWillBeReturned";i:3;s:128:"Tests\Core\Domain\Service\IssueTransactionServiceTest::testThatIfTheAmountIsValidAndTheIssuerHasFundsThenThereWillBeATransaction";i:4;s:121:"Tests\Core\Application\Service\IssueTransactionUseCaseTest::testThatIfTwoUsersExistThereWillBeACallToTheUnderlyingService";i:4;}s:5:"times";a:8:{s:120:"Tests\Core\Application\Service\GetUserBalanceUseCaseTest::testThatIfTheUserIsPresentThenAnSpecificResponseWillBeReturned";d:0.004;s:121:"Tests\Core\Application\Service\GetUserBalanceUseCaseTest::testThatIfNoUserHasBeenFountThenAnSpecificExceptionWillBeThrown";d:0.027;s:103:"Tests\Core\Application\Service\IssueTransactionUseCaseTest::testThatWithNoIssuerTheresAnExceptionThrown";d:0.007;s:105:"Tests\Core\Application\Service\IssueTransactionUseCaseTest::testThatWithNoRecieverTheresAnExceptionThrown";d:0.008;s:121:"Tests\Core\Application\Service\IssueTransactionUseCaseTest::testThatIfTwoUsersExistThereWillBeACallToTheUnderlyingService";d:0.014;s:109:"Tests\Core\Domain\Service\IssueTransactionServiceTest::testThatIfIssuerHasNotFundsThenAnExceptionWillBeThrown";d:0.003;s:111:"Tests\Core\Domain\Service\IssueTransactionServiceTest::testThatIfTheAmountIsNegativeThenAnExceptionWillBeThrown";d:0.003;s:128:"Tests\Core\Domain\Service\IssueTransactionServiceTest::testThatIfTheAmountIsValidAndTheIssuerHasFundsThenThereWillBeATransaction";d:0.004;}}}